src/modules/tree.ts

```diff
@@ -285,6 +285,30 @@ export default {
     ],
     requiredArgs: 'account',
   },
+  e2e: {
+    description: "Run your VTEX app's integration tests",
+    handler: './apps/e2e',
+    options: [
+      {
+        description: 'Specify test request id',
+        long: 'report',
+        short: 'r',
+        type: 'string',
+      },
+      {
+        description: "Test workspace's apps",
+        long: 'workspace',
+        short: 'w',
+        type: 'boolean',
+      },
+      {
+        description: "[Dangerous] Exposes toolbelt token via 'authToken' environment variable",
+        long: 'token',
+        short: 't',
+        type: 'boolean',
+      },
```

Comment:
To change the command name you'll have to change it in this file - see `workspace create` for example